x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/predicate/operator/arithmetic/Mul.java

```diff
@@ -52,4 +52,9 @@ protected Mul replaceChildren(Expression newLeft, Expression newRight) {
     public Mul swapLeftAndRight() {
         return new Mul(source(), right(), left());
     }
+
+    @Override
+    public ArithmeticOperation inverse(Source source, Expression left, Expression right) {
```

Comment:
return `Div` instead of `ArithmeticOperation`.

Reply:
Fixed.